WebContent/js/components/JobInstance.js

```diff
@@ -36,9 +39,25 @@ class JobInstance extends React.Component {
                 this.handleJobToggle(job);
             }
             this.setState({ preventSingleClick: false });
+            this.setState({ keyEnter: 0 });
         }, 500);
     }
 
+    handleKeyDown(e) {
+        const { job } = this.props;
+        if (e.key === 'Enter') {
+            this.setState({ keyEnter: 1 });
+            if (this.state.keyEnter === 0) {
+                this.handleSingleClick(job);
+            } else {
+                this.handleOpenAllFiles(job);
+            }
+        }
+        // else if (e.altKey && e.keyCode === 39) {
```

Comment:
We shouldn't include commented out code which is unused, it will go stale

Reply:
removed, trying out right click didnt work out